fetch.bs

```diff
@@ -6915,9 +7213,16 @@ method steps are:
    <li><p>Set <var>responseObject</var> to the result of <a for=Response>creating</a> a {{Response}}
    object, given <var>response</var>, "<code>immutable</code>", and <var>relevantRealm</var>.
 
+   <li><p>Set <var>responseObject</var>'s <a for=Body>on body consumed</a> to calling
+   <a for=response>finalize and report timing</a> for <var>response</var>,
+   <var>globalObject</var> and "<code>fetch</code>".
+
    <li><p><a for=/>Resolve</a> <var>p</var> with <var>responseObject</var>.
   </ol>
 
+  <p><a>If aborted</a>, then <a for=response>finalize and report timing</a> for <var>response</var>,
```

Comment:
This is not how that Infra construct works. It seems this needs integration with "abort fetch" as well somehow.

Reply:
You marked this as resolved, but https://whatpr.org/fetch/1185.html#dom-global-fetch still has it as far as I can tell.

Reply:
Oops, forgot to erase that line.